src/main/java/org/elasticsearch/indices/memory/IndexingMemoryController.java

```diff
@@ -113,13 +157,14 @@ public IndexingMemoryController(Settings settings, ThreadPool threadPool, Indice
             translogBuffer = ByteSizeValue.parseBytesSizeValue(translogBufferSetting, null);
```

Comment:
not sure I follow - can you unpack? the null here is a default value... not settings name.